packages/main/src/Tokenizer.ts

```diff
@@ -474,9 +475,30 @@ class Tokenizer extends UI5Element {
 			this._expandedScrollWidth = this.contentDom.scrollWidth;
 		}
 
+		this._scrollToEndIfNeeded();
 		this._tokenDeleting = false;
 	}
 
+	_scrollToEndIfNeeded() {
+		const tokenCount = this.tokens.length;
+		if (this._lastTokenCount === undefined) {
+			this._lastTokenCount = tokenCount;
```

Comment:
Initialize the new state properties in the constructor and update the checks here:

```
this._lastTokenCount = 0;
this._lastExpanded = false;
```